client/src/actions/user/index.js

```diff
@@ -1,28 +1,37 @@
 import { UPDATE_USER } from './types'
 
-import userApiClient from '../../api/user'
+import apiOptionsFromState from '../../api/lib/apiOptionsFromState'
+import emailApiClient from '../../api/email'
+import usersApiClient from '../../api/users'
 
 export const updateUser = { type: UPDATE_USER }
 
 export default class UserActionCreator {
-  static applyForProject (project, user) {
-    const projectId = project._id
+  static updating () {}
+
+  static updated () {}
+
+  static failed () {}
```

Comment:
If we're not using `updating`, `updated`, and `failed`, we should not include them for master until we need them.